pkg/agent/testdata/AKSWindows+K8S117/CustomData

```diff
@@ -0,0 +1,488 @@
+[base64(concat('<#
```

Comment:
Did we generate this from aks-engine as baseline?
I think that we should generate data for k8s 1.16. And we also need to generate data for AKS Windows 2004 SIG image and customized image.

Reply:
We also need to generate cse with both sp and msi

Reply:
and customcloud